### ISSUE TITLE
generator ignored GeneratorExit

### ISSUE THREAD

**Issue by bryce-turner** [tgen/jetstream#115]
New issue after updating jetstream_centro to use Jetstream 1.6.
```
[2020-04-03 14:50:08]: LocalBackend spawned(29037): run_pipeline_MMRF_2259_24ed0060a4
[2020-04-03 14:50:10]: Version: 1.6
[2020-04-03 14:51:12]: Version: 1.6
Exception ignored in: <generator object find_pipelines at 0x2aaab97b8eb8>
RuntimeError: generator ignored GeneratorExit
[2020-04-03 14:51:20]: Rendering template...
[2020-04-03 14:51:20]: Template rendering data sources include:
    0: Project: {'__project__': {'args': [...]
    1: Pipeline: {'__pipeline__': {'name': 'phoenix', 'main': 'main.jst', [...]
    2: Command: {}
[2020-04-03 14:51:28]: Loading tasks...
```
I've noticed long running time when running `jetstream pipelines` to get the list of pipelines, but no errors. This does not cause the pipeline to fail, so it is not a critical bug to fix, but it does warrant some investigation.

**Comment by ryanrichholt** [comment]
Thanks Bryce - That should be an easy problem to fix, but I don't imagine it could be causing a long delay in `jetstream pipelines`. That command searches through directories, and so it will be affected by: 1) the number of items in the search path 2) file system performance.

**Comment by ryanrichholt** [comment]
This function needs a try-except to catch the GeneratorExit that is raised when the pipeline is found: 
https://github.com/tgen/jetstream/blob/2cfa6d3c86f3a046456f463fb52cf9909b540201/jetstream/pipelines.py#L144

**Comment by ryanrichholt** [comment]
This line needs to be fixed also, yikes
https://github.com/tgen/jetstream/blob/2cfa6d3c86f3a046456f463fb52cf9909b540201/jetstream/pipelines.py#L166

**Comment by bryce-turner** [comment]
Resolved with PR #133
